apiary.apib

```diff
@@ -3788,6 +3788,15 @@ Get a list of work orders.
                 + location (Address, nullable)
                 + distance (Distance, nullable)
                 + signed: false (boolean)
+                + time (array) - Only included with request parameter `includes=time`.
+                    + (object)
+                        + users (array)
+                            + (object)
+                                + type: `user` (string)
+                                + id: `4ec55a8c-2d80-472a-a9c2-5ff5ee7eac6a` (string)
+                        + started_at: `2019-02-27T10:01:49+00:00` (string)
```

Comment:
Nice to have; also add `ended_at` like we have for time tracking.

Reply:
Seems like a separate feature, I’ll take it up with product